### PR TITLE
small corrections

### PR DESCRIPTION
**Pull request by retokromer**
- no bold
- add comma
- simplify wording
- number the four methods
- stricter link
- note applies on both points

### CODE REVIEW

File: flac.md [hunk 1]
```diff
@@ -80,23 +80,25 @@ Surprisingly, the left-side and right-side forms can be the most efficient in ma
...
+FLAC uses Exponential-Golomb (a variant of Rice) coding as it's residual encoder. You can learn more about [exp-golomb coding](https://en.wikipedia.org/wiki/Exponential-Golomb_coding) on Wikipedia.
 
 FLAC currently defines two similar methods for the coding of the error signal from the prediction stage. The error signal is coded using Exponential-Golomb codes in one of two ways:
 
 1. the encoder estimates a single exp-golomb parameter based on the variance of the residual and exp-golomb codes the entire residual using this parameter;
 
-2. the residual is partitioned into several equal-length regions of contiguous samples, and each region is coded with its own exp-golomb parameter based on the region's mean. (Note that the first method is a special case of the second method with one partition, except the exp-golomb parameter is based on the residual variance instead of the mean.)
+2. the residual is partitioned into several equal-length regions of contiguous samples, and each region is coded with its own exp-golomb parameter based on the region's mean.
```
Comment (dericed) [inline flac.md]: Not `1.`?

Comment (retokromer) [inline flac.md]: Oops! Thank you, will correct.

**Comment by retokromer** [comment]
Sorry, summarising what on the listserv?

**Comment by dericed** [comment]
I mean that sometimes a lot can happen in the repo but the work is not represented in the cellar listserv.

**Comment by retokromer** [comment]
I agree, but I personally don’t see any inconvenience when the work is progressing on GitHub, without having an echo on the listserv.

**Pull request merged**